tests/bluetooth/df/common/src/bt_conn_common.c

```diff
@@ -44,7 +44,7 @@ uint16_t ut_bt_create_connection(void)
 	conn->llcp.cte_req.is_enabled = 0U;
 #endif /* CONFIG_BT_CTLR_DF_CONN_CTE_REQ */
 
-	conn->llcp.fex.features_used |= BIT(BT_LE_FEAT_BIT_CONN_CTE_RESP);
+	conn->llcp.fex.features_used |= BIT(BT_LE_FEAT_BIT_CONN_CTE_REQ);
```

Comment:
Based on the name `BT_LE_FEAT_BIT_CONN_CTE_REQ`, should this be moved inside the `CONFIG_BT_CTLR_DF_CONN_CTE_REQ` guard?

Reply:
I've moved it in a guard, to be sure it is set only when the CTE request is compiled in.